components/encrypt-signup.jsx

```diff
@@ -72,721 +89,722 @@ module.exports = React.createClass({
     return (
       <div className="encrypt-signup">
         {this.props.children}
-        <form id={this.props.formName} onSubmit={this.sendEmailToBasket}></form>
```

Comment:
I believe so, I'm must have hit the 'format' button on my new editor and it handled differently than my old one.